developer_docs/dto/implementation-guidelines.md

```
@@ -235,7 +235,137 @@ facade.findLightsByJpql(jpql, params)     // Missing TemporalType when using Dat
+    + "b.cancelledBill.createdAt, "              // ❌ FAILS SILENTLY if cancelledBill is null
+    + "b.cancelledBill.creater.webUserPerson.name) "  // ❌ FAILS SILENTLY
+    + "FROM Bill b WHERE ...";
+```
+
+**What happens:**
+- If ANY row has `cancelledBill = null`, the ENTIRE query returns 0 results
+- No exception is thrown
+- COUNT query on same data returns correct count (e.g., 1)
+- This is JPQL behavior, not a bug
+
+**✅ SOLUTION 1 (Recommended): Exclude nullable relationship fields from DTO**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)